compiler/modules/bir/verify.bal

```diff
@@ -66,13 +66,24 @@ public function verifyFunctionCode(Module mod, FunctionDefn defn, FunctionCode c
 
 type IntBinaryInsn IntArithmeticBinaryInsn|IntBitwiseBinaryInsn;
 
-function verifyBasicBlock(VerifyContext vc, BasicBlock bb) returns err:Semantic? {
+
+public type InvalidError distinct error;
+
+public function invalidError(d:Message? detail = ()) returns InvalidError {
```

Comment:
This should not be public and should be a method on VerifyContext with a Position argument (same as current `err` method)

Rename the existing err method to semanticErr.  Change each `vc.err("bad BIR: XXX"...)` to `vc.invalidErr("XXX"...)`. Change every other vc.err to vc.semanticErr.